db/views/reporting_patient_blood_sugars_v01.sql

```diff
@@ -0,0 +1,88 @@
+-- Only most recent BP per patient per month. BPs are ordered appropriately below.
+SELECT DISTINCT ON (bs.patient_id, cal.month_date)
+    cal.month_date,
+    cal.month,
+    cal.quarter,
+    cal.year,
+    cal.month_string,
+    cal.quarter_string,
+
+    ------------------------------------------------------------
+    -- Information about the latest BS as of a given month
+    bs.recorded_at AT TIME ZONE 'UTC' AT TIME ZONE 'UTC' AS blood_sugar_recorded_at,
+    bs.id AS blood_sugar_id,
+    bs.patient_id,
+    bs.blood_sugar_type,
+    bs.blood_sugar_value,
+    bs.facility_id AS blood_sugar_facility_id,
+
+    ------------------------------------------------------------
+    -- Risk state of the latest measurement as of given month
+    -- +----------+--------------+------------------+-------------+
+    -- |   Type   | bs-below-200 |  bs-200-to-299   | bs-over-299 |
```

Comment:
Is it possible to translate `bs-below-200`, `bs-200-to-299`, `bs-over-299` to `low`, `medium`, `high` or something similar? It might be confusing to explain why `Hba1c: 9.5` is `bs-over-299` if these values are visible to users.